packages/scandipwa/src/util/Compare/Compare.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * ScandiPWA - Progressive Web App for Magento
+ *
+ * Copyright © Scandiweb, Inc. All rights reserved.
+ * See LICENSE for license details.
+ *
+ * @license OSL-3.0 (Open Software License ("OSL") v. 3.0)
+ * @package scandipwa/base-theme
+ * @link https://github.com/scandipwa/base-theme
+ */
+
+import BrowserDatabase from 'Util/BrowserDatabase';
+
+/**
+ *
+ * @type {string}
+ */
+export const COMPARE_UID = 'compare_uid';
+
+/**
+ *
+ * @type {Number}
+ */
+export const ONE_DAY = 86400;
+
+/**
+ *
+ * @param {String} uid
+ * @returns {void}
+ * @namespace Util/Compare/setUid
+ */
+export const setUid = (uid) => {
+    BrowserDatabase.setItem(uid, COMPARE_UID, ONE_DAY);
+};
+
+/**
+ *
+ * @returns {String/Boolean} uid
```

Comment:
If you really want to use types in jsdoc, use the correct ones.
For this return type, it will be `{string|boolean}`.

Reply:
fix above is included in: https://github.com/scandipwa/scandipwa/pull/2717